### PR TITLE
Docs - Improve SEO

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description

* Added alt attribute to hornet logo
* Fixed yarn warnings
* Added meta tags for description, keywords and image to docs
* Resolves  iotaledger/docs.iota.org#32

## Type of change

- [x] Enhancement (non-breaking change which adds functionality)
- [x] Documentation Fix

# How Has This Been Tested?

Docs site was built locally using node 12.

# Checklist:

- [x] I have made corresponding changes to the documentation
- [x] I have selected the `develop` branch as the target branch

### CODE REVIEW

File: documentation/docs/getting_started/bootstrap_from_a_genesis_snapshot.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
+---
+keywords:
+- genesis snapshot
```
Comment (muXxer) [inline documentation/docs/getting_started/bootstrap_from_a_genesis_snapshot.md:5]: maybe add `bootstrap network` as a keyword as well

File: documentation/docs/post_installation/configuration.md [hunk 2]
```diff
@@ -1,3 +1,14 @@
+---
+keywords:
+- Configuration
+- JSON
+- Customize
+- Interface
```
Comment (muXxer) [inline documentation/docs/post_installation/configuration.md]: I would add `config` and remove `interface`

**Review by muXxer — APPROVED** [review]

File: documentation/docs/getting_started/using_docker_compose.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+- Install
+- Docker Compose
+- Build
+description: Install and run a Hornet node using Docker Compose.
+image: /img/logo/HornetLogo.png
+---
+
 # Using Docker Compose
 
 Make sure that you have read [Using Docker](using_docker.md) before you continue reading as this chapter is for advanced users.  You can find more information on Docker Compose in the [official Docker Compose documentation]( https://docs.docker.com/compose/).
```
Comment (charlesthompson3) [inline documentation/docs/getting_started/using_docker_compose.md]:
```suggestion
Make sure that you have read [Using Docker](using_docker.md) before you continue reading as this chapter is for advanced users.  You can find more information on Docker Compose in the [official Docker Compose documentation](https://docs.docker.com/compose/).
```

File: documentation/docs/post_installation/managing_a_node.md [hunk 4]
```diff
@@ -1,3 +1,14 @@
+---
+keywords:
```
Comment (charlesthompson3) [inline documentation/docs/post_installation/managing_a_node.md:2]: I don't know whether or not it makes an impact but some keywords are capitalized in other sections while some are not. Probably not a big deal, just something I noticed!

File: documentation/docs/post_installation/run_as_a_verifier.md [hunk 5]
```diff
@@ -1,3 +1,13 @@
...
+- Chrysalis Phase 2
+- receipts
+description: How to configure a Hornet node as a verifier node.  
+image: /img/logo/HornetLogo.png
+---
+
+
 # How to Run Hornet as a Verifier Node
 
  A verifier node is a node which validates receipts. Receipts are an integral component of the migration mechanism used to migrate funds from the legacy into the new Chrysalis Phase 2 network. You can find a more detailed explanation on how the migration mechanism works in the [Chrysalis documentation](https://chrysalis.docs.iota.org/guides/migration-mechanism) .
```
Comment (charlesthompson3) [inline documentation/docs/post_installation/run_as_a_verifier.md]:
```suggestion
 A verifier node is a node which validates receipts. Receipts are an integral component of the migration mechanism used to migrate funds from the legacy into the new Chrysalis Phase 2 network. You can find a more detailed explanation on how the migration mechanism works in the [Chrysalis documentation](https://chrysalis.docs.iota.org/guides/migration-mechanism).
```

**Review by charlesthompson3 — APPROVED** [review]
Real minor suggestions, otherwise its solid!

File: documentation/docs/post_installation/managing_a_node.md (hunk 4 above)
Comment (lucas-tortora) [reply documentation/docs/post_installation/managing_a_node.md:2]: I don't think it makes a difference

**Review by huhn511 — APPROVED** [review]
LGTM 👍 

Just thinking to use "IOTA Node" instead (or additional) to "Hornet Node" in the keyword. What do you think?

**Comment by lucas-tortora** [comment]
> LGTM 
> 
> Just thinking to use "IOTA Node" instead (or additional) to "Hornet Node" in the keyword. What do you think?

Since this is going to be hosted on the wiki, I think we need to keep "hornet" as a keyword.  But we could also add IOTA Node.  :)

**Pull request merged**